workflows/phaseimpute/tests/main.nf.test.snap

```diff
@@ -232,17 +232,17 @@
             ],
             "VcfFile [chromosomes=[chr21, chr22], sampleCount=2, variantCount=1739, phased=true]",
             [
-                "NA19401",
-                "NA20359"
+                "NA12878",
+                "NA19401"
             ],
             "VcfFile [chromosomes=[chr21, chr22], sampleCount=2, variantCount=1739, phased=true]",
             [
-                "NA19401",
-                "NA20359"
+                "NA12878",
+                "NA19401"
```

Comment:
Why these changed?

Reply:
I didn't updated it before after fixing the sample id sorting.
This is now in the right order.

Reply:
But not only the order changed, the named did. Before it was: NA19401, **NA20359** and now it is **NA12878**, NA19401.

Reply:
Yeah as the order should be NA12878, NA19401, NA20359 then split in a first file of 2 and a file of 1 sample.